src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -516,21 +558,31 @@ struct GhValenciaDivCleanTemplateBase<
 
    public:
     using factory_classes = tmpl::map<
-        tmpl::pair<DenseTrigger, DenseTriggers::standard_dense_triggers>,
-        tmpl::pair<DomainCreator<volume_dim>, domain_creators<volume_dim>>,
+        tmpl::pair<DenseTrigger,
+                   tmpl::append<DenseTriggers::standard_dense_triggers,
+                                control_system::control_system_triggers<
+                                    control_systems>>>,
+        tmpl::pair<DomainCreator<volume_dim>,
+                   // Currently control systems can only be used with BCO
+                   // domains
+                   tmpl::conditional_t<
+                       use_control_systems,
+                       tmpl::list<::domain::creators::BinaryCompactObject>,
+                       domain_creators<volume_dim>>>,
         tmpl::pair<
             Event,
             tmpl::flatten<tmpl::list<
                 Events::Completion,
-                dg::Events::field_observations<volume_dim, Tags::Time,
-                                               observe_fields,
-                                               non_tensor_compute_tags>,
+                dg::Events::field_observations<
+                    volume_dim, Tags::Time, observe_fields,
```

Comment:
How does this compile? `Tags::Time` should be the list of fields when I look at the definition. What am I missing?